corehq/apps/registration/views.py

```diff
@@ -269,44 +268,52 @@ def post(self, request, *args, **kwargs):
         nextpage = request.POST.get('next')
         form = DomainRegistrationForm(request.POST)
         context = self.get_context_data(form=form)
-        if form.is_valid():
-            reqs_today = RegistrationRequest.get_requests_today()
-            max_req = settings.DOMAIN_MAX_REGISTRATION_REQUESTS_PER_DAY
-            if reqs_today >= max_req:
-                context.update({
-                    'current_page': {'page_name': _('Oops!')},
-                    'error_msg': _(
-                        'Number of domains requested today exceeds limit (%d) - contact Dimagi'
-                    ) % max_req,
-                    'show_homepage_link': 1
-                })
-                return render(request, 'error.html', context)
+        if not form.is_valid():
+            return self.render_to_response(context)
 
-            try:
-                domain_name = request_new_domain(request, form, is_new_user=self.is_new_user)
-            except NameUnavailableException:
-                context.update({
-                    'current_page': {'page_name': _('Oops!')},
-                    'error_msg': _('Project name already taken - please try another'),
-                    'show_homepage_link': 1
-                })
-                return render(request, 'error.html', context)
+        if settings.RESTRICT_DOMAIN_CREATION and not request.user.is_superuser:
+            context.update({
+                'current_page': {'page_name': ('Oops!')},
+                'error_msg': ('Your organization has requested that domain creation should be restricted. '
+                              'For more information, please speak to your administrator.'),
```

Comment:
This error should be wrapped in a `_()` tag for translation, and I think it should say "project" rather than "domain".  (I do see that one of the existing messages also refers to domains, would you update that while you're making these changes?)

Reply:
Ah, I was wondering what that syntax was about. Thanks